CODE_OF_CONDUCT.nl.md

```diff
@@ -0,0 +1,140 @@
+van een incident te respecteren.
+
+## Handhavingsrichtlijnen
+
+Communityleiders zullen onderstaande "Community Impact Guidelines" volgen bij
+het bepalen van de consequenties van elke handeling die zij in strijd met de
+Gedragscode achten:
+
+### 1. Correctie
+
+**Impact op de community**: Gebruik van ongepast taalgebruik en ander gedrag dat
+onprofessioneel of als niet gewenst wordt geacht in de community.
+
+**Consequentie**: Een persoonlijke, schriftelijke waarschuwing van
+communityleiders, verduidelijking over de aard van de overtreding en een uitleg
+waarom het gedrag ongepast wordt geacht. Mogelijk wordt hierbij om een openbare
+verontschuldiging gevraagd.
+
+### 2. Waarschuwing
+
+**Impact op de community**: Een overtreding door een enkel incident of reeks van
+acties.
+
+**Consequentie**: Een waarschuwing met gevolgen als het gedrag voortduurt. Geen
+interactie met de betrokken personen, inclusief ongevraagde interactie met
+degenen die de Gedragscode handhaven, voor een bepaalde periode. Dit omvat het
+vermijden van interacties in gemeenschappelijke ruimtes en externe kanalen zoals
+op social media. Het overtreden van deze voorwaarden kan leiden tot een
+tijdelijke of permanente ban.
+
+### 3. Tijdelijk verbod
+
+**Impact op de community**: Een ernstige schending van community normen,
+inclusief aanhoudend ongepast gedrag.
+
+**Consequentie**: Een tijdelijk verbod op elke vorm van interactie of openbare
+communicatie met de community gedurende een bepaalde periode. Geen publieke- of
+privé-interactie met de betrokken personen, inclusief ongevraagde interactie met
+degenen die de Gedragscode handhaven, is toegestaan tijdens deze periode. Het
+overtreden van deze voorwaarden kan leiden tot een permanente ban.
+
+### 4. Permanente ban
+
+**Impact op de community**: Een patroon van schending van de community
+standaarden vertonen, waaronder aanhoudend ongepast gedrag, intimidatie van een
+individu of agressie jegens- of kleinering van groepen van individuen.
+
+**Consequentie**: Een permanent verbod op elke vorm van openbare interactie in
+de community.
+
+## Attributie
+
+Deze gedragscode is een aangepaste versie van de [Contributor
+Covenant][homepage], versie 2.0, beschikbaar op
+[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].
+
+Richtlijnen voor impact op de community zijn geïnspireerd door [Mozilla's code
+of conduct enforcement ladder][mozilla coc].
+
+Voor antwoorden op veelgestelde vragen over deze gedragscode, zie de
+veelgestelde vragen op [https://www.contributor-covenant.org/faq][faq]. Er zijn
+vertalingen beschikbaar op
+[https://www.contributor-covenant.org/translations][translations].
+
+[homepage]: https://www.contributor-covenant.org
+[v2.0]: https://www.contributor-covenant.org/version/2/0/code_of_conduct.html
+[mozilla coc]: https://github.com/mozilla/diversity
+[faq]: https://www.contributor-covenant.org/faq
+[translations]: https://www.contributor-covenant.org/translations
```

Comment:
Waar komen deze linkjes onderin vandaan?

Reply:
Ik heb de CoC gekopieerd uit een van de andere repos. Zie bijvoorbeeld: https://github.com/nl-design-system/documentatie/blob/main/CODE_OF_CONDUCT.nl.md?plain=1#L83-L87

Blijkbaar is er een regel die graag ziet dat er voor die [reference links](https://github.github.com/gfm/#reference-link) lowercase gebruikt wordt. In de `documentatie` repo is het nog `[FAQ]` maar hier `[faq]`.

Reply:
Deze linkjes onderin moeten weg, het lijkt me een foutje dat ze erin zijn gekomen.

Reply:
Wil je dat ik de reference links aanpas naar inline links dan of begrijp ik het verkeerd en moet die hele attributie paragraaf weg?

https://github.com/nl-design-system/example-with-next.js/blob/a59e8eecc98b309b8865719e30ac445c3a27480d/CODE_OF_CONDUCT.nl.md dit is hoe hij gerenderd wordt nu.